### PR TITLE
Introducing the vastly improved voice alert and voice menu feature

### PR DESCRIPTION
**Pull request by TheRealMoeder**
based on the initial work by MWM and phantom-8

### CODE REVIEW

File: src/extended_audio.c [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+#include <libopencm3/stm32/usart.h>
+#endif // EMULATOR
+
+#if HAS_EXTENDED_AUDIO
+
+static u32 audio_queue_time = 0;
+
+// Initialize UART for extended-audio
+void AUDIO_Init() {
+#ifdef BUILDTYPE_DEV
```
Comment (victzh) [inline src/extended_audio.c]: You can omit #ifdef BUILDTYPE_DEV - in non dev builds printf is an empty macros so it's totally eliminated and is not present in the final binary at all.

Comment (TheRealMoeder) [inline src/extended_audio.c]: I thought it was not eliminated, since many protocols use a dbgprintf which is redefined...

Comment (TheRealMoeder) [inline src/extended_audio.c]: Just rechecked std.h, you are of course right...I will remove the ifdefs

**Comment by TheRealMoeder** [comment]
what do you suggest?

**Pull request merged**